CHANGES.md

```diff
@@ -24,6 +24,13 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx] ###
 
+ * 'Configure' has been changed to figure out the configuration target if
+   none is given on the command line.  Consequently, the 'config' script is
+   now only a mere wrapper.  All documentation is changed to only mention
+   'Configure'.
+   
+   *Richard Levitte*
```

Comment:
personal references in the comments shouldn't occur

Reply:
Er...  what???  This is CHANGES.md, we have personal references in each and every entry!

Reply:
BTW they are done with []'s - I missed the context in the review ...

Reply:
That got changed when CHANGES was converted to Markdown

Reply:
I think my name should also be here FWIW.

Reply:
You're absolutely right.  Will do